src/main/java/seedu/duke/userinterface/command/Add.java

```diff
@@ -1,8 +1,48 @@
 package seedu.duke.userinterface.command;
 
+import seedu.duke.exceptions.IncorrectDeadlineFormatException;
+import seedu.duke.exceptions.TaskTitleException;
+import seedu.duke.exceptions.TaskWrongFormatException;
+import seedu.duke.tasks.Task;
+import seedu.duke.tasks.TaskList;
+import seedu.duke.userinterface.CliMessages;
+import seedu.duke.userinterface.InputParser;
+
 public class Add extends CliCommand {
+    public static final String COMMAND_WORD = "add";
```

Comment:
We could also potentially add this as a field in CliCommand, do you think that might be a good idea? Since every command will have a command word.

Reply:
Yea sure, that would be nice.